packages/thunderstore-api/src/errors.ts

```diff
@@ -3,17 +3,22 @@ export function isApiError(e: Error | ApiError | unknown): e is ApiError {
 }
 
 export class ApiError extends Error {
-  constructor(message?: string) {
+  response: { [key: string]: string[] } | undefined;
```

Comment:
Calling this `responseData` or such would be more appropriate. `response` would be appropriate if you stored the actual response object itself.